### ISSUE TITLE
Unable to move pipeline group when using pipeline filter

### ISSUE THREAD

**Issue by chris-gillatt** [gocd/gocd#12427]
##### Issue Type
- Bug Report

##### Summary

When on the 'Pipelines' screen, users are not able to move pipelines to other pipeline groups when using the search filter box.  It's a minor annoyance in production for our administrators where we have many pipelines and pipeline groups.

##### Environment

We run GoCD in RockyLinux 9 Docker containers on k8s. 

###### Basic environment details

* Go Version: `23.5.0-18179`
* JAVA Version: `17.0.9`
* OS: `Linux 5.15.120+` (Rocky Linux 9.3)
* Browser vendor and version: bug appears on latest versions of Firefox, Safari and Chrome. 

I've tested this locally in versions from 22.3.0 to latest - 23.5.0 and I have a feeling that this has probably been an issue for a long time.  I could not find an existing issue, but apologies if this is a duplicate.

##### Steps to Reproduce

1. Select `Admin` from the top menu, then`Pipelines`
2. Use the `search for a pipeline name` input box to search for a pipeline that exists in any group
3. Try and Move Pipeline to another group (move button becomes greyed out)

##### Expected Results
User should be able to select a new pipeline group to move the pipeline to. 

##### Actual Results
Move button becomes greyed out, and the move functionality disabled.  I believe this is because the move button is reading from the search filter, which is not much use.  The move button should not be limited by the search filter. 

##### Possible Fix
Workaround: use the browsers' built-in find (control/command+f) instead of the GoCD GUI search box.

##### Code snippets/Screenshots
https://github.com/gocd/gocd/assets/9963006/dd2479b7-fae1-41ae-b4f0-9b9678856499

**Comment by chadlwilson** [comment]
Thanks for reporting, Ive also noticed this.

Any enthusiasm to see if there's a good reason for this and propose a change (or a PR?)?

**Comment by chris-gillatt** [comment]
Yeah we can take a look in the near future unless someone beats us to it.

**Comment by chris-gillatt** [comment]
@jonathan-dalton @balkrishna-gukhool 👀
